pkg/pillar/types/global.go

```diff
@@ -948,6 +952,7 @@ func NewConfigItemSpecMap() ConfigItemSpecMap {
 	// LogRemainToSendMBytes - Default is 2 Gbytes, minimum is 10 Mbytes
 	configItemSpecMap.AddIntItem(LogRemainToSendMBytes, 2048, 10, 0xFFFFFFFF)
 	configItemSpecMap.AddIntItem(DownloadMaxPortCost, 0, 0, 255)
+	configItemSpecMap.AddIntItem(BlobDownloadMaxRetries, 5, 1, 10)
```

Comment:
@jsfakian , I guess you come up with 5 attempts as a good value from your tests, but I'm wondering if 10 would be enough on very poor networks.... is there any specific reason for this maximum range?

Reply:
There is no specific reason for the maximum of ten retries; we change it to infinity, but ten (or pick any number) retries should be enough to download the image successfully. Otherwise, we need to investigate why it failed so many times.

Reply:
Please, don't change it to maximum integer, my point was whether 10 (and not 20 or 50) was enough to download the image or not.... I was trying to understand the context.... if you notice that 10 attempts, in general, it's enough, then that's fine. I would use 10x the number of retries that you observed in your tests. For instance, if you notice that the downloader is doing 5 or 6 retries, then I would set the maximum to 50 or 60. However, if you notice that only 2 or 3 retries are made, then 10 should be fine....

Reply:
The maximum number of retries is not very important. Currently, we immediately retry the download if the verification fails. We might need to add a delay that exponentially grows each time we retry.